exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -196,7 +196,7 @@ def _create_database_info(self, db_ip_address: str, reused: bool) -> DatabaseInf
             ssh_info = SshInfo(self.ssh_user, self.ssh_port_forward, self.ssh_key_file)
             database_info = DatabaseInfo(
                 host=db_ip_address,
-                ports=Ports.default_ports,
+                ports=self.ports,
```

Comment:
this way, you don't expose the port forwards, but the default ports

Reply:
I am not sure whether this code actually _exposes_ the ports. 
In my understanding this code just creates the `database_info`.
I must admit that I actually don't know for which purposes the `database_info` is used.
The implementation up to ITDE version 1.7.1 used independent constants here:
```python
BUCKETFS_PORT = "6583"
DB_PORT = "8888"
DatabaseInfo(host=db_ip_address, db_port=DB_PORT, bucketfs_port=BUCKETFS_PORT,
                  reused=reused, container_info=container_info)
```
but did not put the forwarded ports into database_info.
Let's discuss, as soon as you're ready.

Reply:
Ok, my explanation wasn't clear. The question, who uses the DatabaseInfo and for what. The ITDE uses in it's other Tasks to access the DB. The user on the host might use it to access the DB (port forward) and also the user in the application container might use it. The issue is actually, that the host has a different views (port forward), the other two use the default ports. I think, the solution is actually to add an attribute for the port forward to the DatabaseInfo. This way we can accommodate both views

Reply:
I will add both groups of ports: internal as well as forwards

Reply:
Ports added in latest pull request